### ISSUE TITLE
HATS Web: Form Builder Push from QA to Prod

### ISSUE THREAD

**Issue by joeschmit** [WSDOT/HATS#1333]
Admin feature: ability to push a form from QA to Prod once we have tested it. Some forms are getting "complicated" and it would be efficient to have the ability to push to prod once satisfied in QA.

**Comment by brianblood** [comment]
That's a fine idea.

**Comment by jhudsonWA** [comment]
If we did this I would suggest an export and import type feature. The ids and such don’t really match between environments. But we could create a process that parses a json or xml file that recreates the form.

Sent from my iPhone

On Jun 10, 2019, at 12:45 PM, Joe Schmit <notifications@github.com<mailto:notifications@github.com>> wrote:


Admin feature: ability to push a form from QA to Prod once we have tested it. Some forms are getting "complicated" and it would be efficient to have the ability to push to prod once satisfied in QA.
